.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2.1
+
+jobs:
+  build:
+    machine:
+      image: ubuntu-2004:current
+    resource_class: medium
+    steps:
+      - run:
+          name: "Install build system dependencies"
+          command: |
+            sudo apt-get update
+            sudo apt-get install python3 python3-setuptools python3-pip wget
+            python3 -m pip install --user build
+      - checkout
+      - run:
+          name: "Build s1-reader python package"
+          command: |
+            python3 -m build --sdist --wheel .
```

Comment:
Thanks @rtburns-jpl. s1-reader depends on isce3 . The rest of dependencies are [here](https://github.com/opera-adt/s1-reader/blob/main/requirements.txt). As we discussed we want to move forward with isce3 on conda package.